tests/integration/test_fim/test_files/test_ambiguous_confs/test_ignore_works_over_restrict.py

```diff
@@ -90,7 +90,9 @@
 # Configurations
 
 conf_params, conf_metadata = generate_params()
-configurations = load_wazuh_configurations(configurations_path, __name__, params=conf_params, metadata=conf_metadata)
+configurations = load_wazuh_configurations(
+    configurations_path, __name__, params=conf_params, metadata=conf_metadata
+)
```

Comment:
No, it was because of the formatter I ran. It was restored as it was previously in commit [b6dcc84](https://github.com/wazuh/wazuh-qa/pull/3078/commits/b6dcc84036d83f6326e7f8757d7addb4e3c94de7)